wpeverest/everest-forms
# Feature - Multiple choices/options selection

### All Submissions:






### Changes proposed in this Pull Request:



This PR aims to support multiple option selection in either normal dropdown or enhanced select2 dropdown.

Additionally, this PR aims to patch two issues:

1. The default value for the dropdown is broken.
2. If enhanced select is present, on frontend required message is moved just above the field.

### How to test the changes in this Pull Request:

1. Create a form and add a dropdown field before checkout to this branch.
2. If you are on a free plan, you should be preview the feature upgrade modal.
3. Switch to (Pro) plan and choose **Multiple selections** just below the choices.
4. Check if the old dropdown field has no compatibility issues. Again add the new dropdown field and similar steps!
5. Also, make sure above both additional patches are working smoothly!
6. Make sure field options are perfectly compatible as always with the new changes.
7. Next, the editable entry should be okay for both normal and enhanced select.
8. I haven't touched the conditional logic part but at least let's not miss this because if something is changed it is being affected!

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Add support for multiple option selection in the Pro plan.
> Fix - The default value for a dropdown field not set.
> Fix - Enhanced select required message shown above field.
